lciafmt/iw.py

```diff
@@ -5,10 +5,16 @@
 import lciafmt.df as df
 import lciafmt.util as util
 
-print([x for x in pyodbc.drivers() if x.startswith('Microsoft Access Driver')])
+# Check for drivers and display help message if absent
+Driver_Check = ([x for x in pyodbc.drivers()])
+if any('Microsoft Access Driver' in word for word in Driver_Check):
+    print("Drivers Available")
+else:
+    print("Please install drivers to remotely connect to Access Database. Drivers only available on windows platform. "
+          "For instructions visit: https://github.com/mkleehammer/pyodbc/wiki/Connecting-to-Microsoft-Access")
 
+# Download Access file and call read function to transfer into dataframe
```

Comment:
Function descriptions should be included below the function name using (line 17) so that they can be read by interpreters